src/main/java/console_app/ApplicationDriver.java

```diff
@@ -341,6 +343,60 @@ private static boolean isInt(String string) {
     }
 
 
+    private DatesForm createForm() {
+
+        String[] daysOfWeek = {"monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"};
+        Set<String> daysSet = new HashSet<>(Arrays.asList(daysOfWeek));
+
+        Scanner sc = new Scanner(System.in);
+        String input;
+        do {
+            System.out.println("Input what day of the week you want the event to reoccur");
+            input = sc.nextLine().toLowerCase();
+        } while (!daysSet.contains(input));
+
+        int index = 0;
+        while (!daysOfWeek[index].equals(input))
+            index++;
+
+        DayOfWeek day = DayOfWeek.of(index + 1);
+
+        System.out.println("When in the day do you want the event to reoccur (HH:mm)?");
+        Scanner scanner = new Scanner(System.in);
+        String timeString = scanner.nextLine();
+        LocalTime timeOfDay = LocalTime.parse(timeString);
+
+        System.out.println("(Optional) input the time to start from (enter nothing to not use)");
```

Comment:
What's the difference between this and the one above (when in the day do you want the event to reoccur)?

Reply:
or is this and the one below duration?

Reply:
Aha good question. It's supposed to be when the Weekly occurrences should start from/end by. E.g. Every Monday at 13:00 from September 1 until December 31.

I can make the messages clearer.

Reply:
ah I see. Yea clearer message would be good, I also think including a format thing like you did above would be helpful too. Like when you added (HH:mm) to the prompt above this one.